core-services/prow/02_config/openshift/console/_pluginconfig.yaml

```diff
@@ -33,6 +33,14 @@ label:
       assign_on:
       - label: backport-risk-assessed
       label: cherry-pick-approved
+    - allowed_users:
+      - jhadvig
+      - rhamilto
+      - spadgett
+      - TheRealJon
+      - kmcfaul
+      - nicolethoen
```

Comment:
I think we'd also want to add @vikram-raj and @rohitkrai03 from ODC.

@vikram-raj @rohitkrai03 We plan to hold other non-critical changes until the PF6 PR merges, so we're adding a label.

Reply:
added both